presto-resource-group-managers/src/main/java/com/facebook/presto/resourceGroups/SelectorSpec.java

```diff
@@ -42,6 +44,7 @@ public SelectorSpec(
             @JsonProperty("clientTags") Optional<List<String>> clientTags,
             @JsonProperty("selectorResourceEstimate") Optional<SelectorResourceEstimate> selectorResourceEstimate,
             @JsonProperty("clientInfo") Optional<Pattern> clientInfoRegex,
+            @JsonProperty("catalog") Optional<String> schema,
```

Comment:
```suggestion
            @JsonProperty("schema") Optional<String> schema,
```

BTW should we add catalog support as well?

Reply:
Yes we can do that. But for this one want to keep it to schema. We want to use that for our purpose :)

Reply:
let me send a followup PR to support catalog.